fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/leaderboard/LeaderboardIndexTest.java

```diff
@@ -527,6 +530,52 @@ public void basicUngroupedNestedReverse() {
         }
     }
 
+    @ParameterizedTest(name = "emptyUngroupedNested[highScoreFirst={0}]")
+    @BooleanSource
+    public void emptyUngroupedNested(boolean highScoreFirst) {
```

Comment:
This test actually passes both before and after this change, but having coverage of an empty index seemed like a good idea.